rms/file_processing/managers.py

```diff
@@ -0,0 +1,19 @@
+from rms.file_processing.models import File
+from rms.utils.managers import BaseModelManager
+from fastapi import UploadFile
+from abc import ABC, abstractmethod
+from rms.settings import Settings
+
+
+class FileManager(BaseModelManager[File]):
+    __model__ = File
+
+
+class StorageClient(ABC):
+    def __init__(self):
+        self.settings = Settings()
+
+    @abstractmethod
+    async def upload(self, file: UploadFile, file_name: str) -> str:
+        """Upload a file and return its PATH."""
+        pass
```

Comment:
This was supposed to be moved to the clients file, I am a dumb dumb and copied the wrong path sry :<

Reply:
fixed